indico/htdocs/sass/modules/_designer.scss

```diff
@@ -352,3 +352,24 @@
 .template-side.back:not(.empty) {
     pointer-events: none;
 }
+
+.toggle-default {
+    opacity: 0.3;
+
+    &.default-on-category {
+        opacity: 1;
+        color: $blue;
+
+        &.inherited-from-root {
+            opacity: 0.6;
+
+            &:hover {
+                color: $blue;
```

Comment:
Well this is in case of default template inherited from root category which cannot be unclicked so I didn't want it to turn black on hover so it doesn't seem like you can click it